Cargo.toml

```diff
@@ -234,6 +234,10 @@ sha2 = { version = "0.10.8", default-features = false, optional = true }
 # GreptimeDB
 greptimedb-client = { git = "https://github.com/GreptimeTeam/greptimedb-ingester-rust.git", rev = "4cb19ec47eeaf634c451d9ae438dac445a8a3dce", optional = true }
 
+# Sentry
+sysinfo = "*"
```

Comment:
do we still need that?

Reply:
Nope! Gone in 95fe67080490fe9ebf52e76262dbddff3b69759c.